packages/mongodb-chatbot-server/src/processors/makeVerifiedAnswerGenerateUserPrompt.ts

```diff
@@ -0,0 +1,44 @@
+import { FindVerifiedAnswerFunc } from "./FindVerifiedAnswerFunc";
+import {
+  GenerateUserPromptFunc,
+  GenerateUserPromptFuncReturnValue,
+} from "./GenerateUserPromptFunc";
+
+export interface MakeVerifiedAnswerGenerateUserPromptParams {
+  /**
+    Find content based on the user's message and preprocessing.
+   */
+  findVerifiedAnswer: FindVerifiedAnswerFunc;
+
+  continuation: GenerateUserPromptFunc;
+}
+
+export const makeVerifiedAnswerGenerateUserPrompt = ({
```

Comment:
think a doc comment on top of the constructor function would be good too. 

high level overview of what this GenerateUserPrompt func does. namely:

1. try to find verified answer for user query
2. if cannot find, proceed to use the other GenerateUserPrompt func